packages/timeline-state-resolver/src/conductor.ts

```diff
@@ -1492,3 +1533,43 @@ function removeParentFromState(o: TimelineState): TimelineState {
 	}
 	return o
 }
+
+/**
+ * If aborted, rejects as soon as possible, but lets the wraped function safely resolve or reject on its own
```

Comment:
I think there is a mismatch between this documentation and the method name.
The method name indicates that you have something that knows nothing about abortion logic and then make it able to be aborted.
The documentation indicates that whatever is given to this method allready know whether or not is aborted i.e. it is allready abortable.

Reply:
I agree, but what other name do you suggest? I have no good ideas besides `makeImmediatelyAbortable`, because what it receives is _normally_ or rather _poorly_ abortable, whereas this makes it kind of immediate

Reply:
I don't have a good suggestion